products/jbrowse-web/src/StartScreen.tsx

```diff
@@ -109,6 +109,7 @@ export default function StartScreen({
   const [sessions, setSessions] = useState<Record<string, any>>()
   const [sessionToDelete, setSessionToDelete] = useState<string>()
   const [sessionToLoad, setSessionToLoad] = useState<string>()
+  const [userMessage, setUserMessage] = useState<string>()
```

Comment:
often use <unknown> as the type when passing the raw error as we don't necessarily know if the error is a Error object, and then @jbrowse/core/ui/ErrorMessage stringifies it as needed